contrib/node/src/python/pants/contrib/node/targets/node_module.py

```diff
@@ -49,6 +49,9 @@ def __init__(
                            E.G. { 'app': './cli.js', 'runner': './scripts/run.sh' }
                            `string`, file path and package name as  the default bin name
                            E.G. './cli.js' would be interpreted as { 'app': './cli.js' }
+    :param node_scope: Groups related packages together by adding a scope. The `@`
+      symbol is typically used for specifying scope in the package name in `package.json`.
+      However pants target addresses do not allow for `@` in the target address.
```

Comment:
This should probably refer to the `--node-distribution-node-scope` option, and how defaulting works.